### PR TITLE
Visual Improvements

### LINKED ISSUE #661: Visual Improvements
We should address the following visual improvements:

- [x] Improve OR/AND selector in the "Gitcoin Passport" census type. Change the current switch to the "custom radio button" Chakra component, while preserving the text change depending on the selected option: 
image
- [x] In the "Gitcoin passport" census, once selected add in the end of the description the following text "Read more about Gitcoin's Scorer & Thresholds here" with a link in the "here" that opens a new tab linking: "https://docs.passport.gitcoin.co/building-with-passport/major-concepts/scoring-thresholds"
- [x] The minimum possible GPS should be "1" instead of "0" (both for the numeric input & slider). 
- [x] The default GPS should be set to "20".
- [x] Correct the typo: `sybyl-attack` to `sybil-attack`.
- [x] In the "Token census", the description is as follows: "Configure the census by choosing a token from any available network. You can request a token here." but the "here" is not a link. Add the link to the "Add token form" (https://tally.so/r/mO46VY).
- [x] Change the text of the modal:
image
By removing: "Option not available in the free plan. To use it, you need to purchase the custom project plan." and replacing by: "XXXX is a Premium feature". (for instance: `Customization is a Premium Feature` or `SMS-based census is a Premium Feature`). 
Also, change the "Or you can directly schedule one quick call (30'') with our experts to help you create your desired voting project." to "Schedule a 30-minute call with our team to discuss your voting project. We'll guide you through the process and tailor solutions to meet your specific needs."
Same text for Spanish and Catalan.
- [x] Remove top margin space from the "Verify your vote" box and add some bottom margin.
image
- [x] Change "You can correct your vote only once." to "You can correct your vote once."
- [x] Add the "magic" emoji in the phrase: "Please wait while the anonymous magic happens  🪄."
- [x] When click in the "Request Custom Tokens", the form is open but also the option is selected in the dropdown. Prevent the dropdown to show this as an option: 
image
- [x] The "pro" label in the census is not shown correctly (onvote theme): 
image
- [x] Add spacing between the title and the description of the "voting types" & "other settings" cards: 
image
image

### PR DESCRIPTION
**Pull request by selankon**
Fix #661 

From the list it fixes 

- [x] In the "Gitcoin passport" census, once selected add in the end of the description the following text "Read more about Gitcoin's Scorer & Thresholds here" with a link in the "here" that opens a new tab linking: "https://docs.passport.gitcoin.co/building-with-passport/major-concepts/scoring-thresholds"
- [x] The minimum possible GPS should be "1" instead of "0" (both for the numeric input & slider). 
- [x] The default GPS should be set to "20".
- [x] Correct the typo: `sybyl-attack` to `sybil-attack`.
- [x] In the "Token census", the description is as follows: "Configure the census by choosing a token from any available network. You can request a token here." but the "here" is not a link. Add the link to the "Add token form" (https://tally.so/r/mO46VY).
- [x] Remove top margin space from the "Verify your vote" box and add some bottom margin.
image
- [x] Change "You can correct your vote only once." to "You can correct your vote once."
- [x] Add the "magic" emoji in the phrase: "Please wait while the anonymous magic happens  🪄."
- [x] When click in the "Request Custom Tokens", the form is open but also the option is selected in the dropdown. Prevent the dropdown to show this as an option: 
image

### CODE REVIEW

File: src/i18n/locales/ca.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "votes_one": "<span>1</span> <text>votant</text>",
     "votes_many": "<span>{{ count }}</span> <text>votants</text>",
     "votes_other": "<span>{{ count }}</span> <text>votants</text>",
-    "voting_anonymous_advice": "Si us plau, espera mentre la màgia té lloc.Això pot trigar uns minuts; no tanquis la finestra."
+    "voting_anonymous_advice": "Si us plau, espera mentre la màgia té lloc \uD83E\uDE84.\nAixò pot trigar uns minuts; no tanquis la finestra."
```
Comment (elboletaire) [inline src/i18n/locales/ca.json:14]: Why not simply use the emoji 🪄?

**Review by elboletaire — CHANGES_REQUESTED** [review]
I think the margin needs to be fixed. I'm also wondering why use hex values instead of just the emoji, which is more human-readable.

**Comment by selankon** [comment]
> IMHO some more bottom margin should be added to compensate the top margin (or removed from the top.. either way is ok)

Oh, I forgot to increase the margin bottom on the onvote theme... Does this enough?

image

File: src/i18n/locales/ca.json (hunk 1 above)
Comment (selankon) [reply src/i18n/locales/ca.json]: When I copied the emoji to the json the IDE just pasted the hex, don't know why... I just pushed a fix 566632e

**Review by elboletaire — APPROVED** [review]

**Pull request merged**